partiql-lang/build.gradle.kts

```diff
@@ -75,6 +75,18 @@ dependencies {
     }
 }
 
+apiValidation {
+    /**
+     * Classes (fully qualified) that are excluded from public API dumps even if they
+     * contain public API.
+     */
+    ignoredClasses.addAll(
+        listOf(
+            "org.partiql.lang.compiler.PartiQLCompilerPipeline" // deprecated
+        )
+    )
+}
```

Comment:
Example configuration for a specific package to ignore a certain, deprecated class. The Kotlin binary API compatibility checker provides some more configurability here -- https://github.com/Kotlin/binary-compatibility-validator/tree/master?tab=readme-ov-file#optional-parameters.